gradle.properties

```diff
@@ -2,7 +2,7 @@ minecraft_version=1.7.10
 forge_version=10.13.4.1614-1.7.10
 forgeDep_version=10.13.4.1614
 
-mod_version=2.3.1.16_beta
+mod_version=2.3.1.17_beta
```

Comment:
Maybe we should remove beta suffix? As in from user perspective it's a mod which exits for a long time and works good enough in "prod"

Reply:
Ok, I've removed it